src/lib/mod.js

```diff
@@ -12,7 +12,23 @@
  * sanitized modulus function that always returns in the range [0, d)
  * rather than (-d, 0] if v is negative
  */
-module.exports = function mod(v, d) {
+function mod(v, d) {
     var out = v % d;
     return out < 0 ? out + d : out;
+}
+
+/**
+ * sanitized modulus function that always returns in the range [-1.5*d, 1.5*d]
+ * rather than (-d, 0] if v is negative
```

Comment:
Behavior looks correct, but shouldn't the doc be:
```
sanitized modulus function that always returns in the range[-d, d]
rather than (-2*d, 0] if v is negative and [0, d) if v is positive
```
That said, since "half" is in the name I might have used d2 as the argument - since that's still in the end the value we're shifting by... not a big deal though, if it makes more sense to you this way feel free to keep it.

Reply:
> I might have used d2 as the argument

Good call. Done in https://github.com/plotly/plotly.js/pull/2954/commits/d2484f39265223943f199658500f2218e9eb929b

Bonus that made the doc correct.

Reply:
> that made the doc correct

Almost :neckbeard: 
`[-1.5*d, 1.5*d]` -> `[-0.5*d, 0.5*d]` or `[-d/2, d/2]`